packages/amplify-cli/src/extensions/amplify-helpers/write-json-to-dart.js

```diff
@@ -0,0 +1,15 @@
+import fs from 'fs-extra';
+import path from 'path';
+
+function writeJsonToDart(dest, obj) {
+  const destPath = path.parse(dest).dir;
+  if (!fs.existsSync(destPath)) {
+    fs.mkdirSync(destPath, { recursive: true });
+  }
+  const dartContent = `const amplifyconfig = ''' ${obj}''';`;
```

Comment:
Any particular reason we are creating a dart file with the exports existing a string instead of writing them to a json file?

Reply:
Yeah - for easier importation into Flutter project.